### PR TITLE
Update kubernetes godep dependency

### PR DESCRIPTION
**Pull request by mvdan**
Many changes in kubernetes - had to fix a few imports to adapt to kubernetes 1.1.0-alpha.

In particular, I need this godep bump to use the new `unversioned.Time` as opposed to `util.Time`.

### CODE REVIEW

**Comment by mvdan** [comment]
CC @vishh

**Comment by mvdan** [comment]
@vishh all green, PTAL

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,12 +1,16 @@
 {
 	"ImportPath": "k8s.io/heapster",
-	"GoVersion": "go1.5",
+	"GoVersion": "go1.5.1",
```
Comment (vishh) [inline Godeps/Godeps.json]: Was this intentional?

**Comment by vishh** [comment]
LGTM. Just one comment. Feel free to merge once that is resolved.

**Pull request merged**